src/createServer.js

```diff
@@ -1,9 +1,35 @@
 /* eslint-disable no-console */
 'use strict';
 
+const http = require('http');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    const [pathname, queryOption] = req.url.split('?');
+    const parts = pathname
+      .slice(1)
+      .split('/')
+      .filter((part) => part !== '/' && part !== '');
```

Comment:
The `filter` function is filtering out empty strings, which is correct, but the condition `part !== '/'` is unnecessary because `split('/')` will never produce a part that is just '/'. You can simplify the filter condition to `part !== ''`.